concourse/steps/notification.mako

```diff
@@ -249,4 +249,20 @@ mailutil.notify(
     email_cfg_name=email_cfg_name,
     recipients=email_cfg['recipients'],
 )
+slack_cfg = ${on_error_cfg.slack()}
+if slack_cfg:
+  channel_cfg = slack_cfg.get('channel_cfgs')[0]
```

Comment:
if we allow a list of channel-cfgs (which seems like a good idea), then I think we should also honour each cfg (as opposed to hardcoding usage of first element and silently ignoring remainder).

btw, rather use if on template-level (or early-exit) to avoid indentation level, like so:
```
% if on_error_cfg.slack():
<code for sending notifications>
% endif
```

this has the additional (small) benefit of rendering less (unused) code